dfm/cleaning/quality.py

```diff
@@ -275,3 +279,16 @@ def stop_word(doc: Doc, n: int, stop_words: set) -> bool:
         """
         n_stopwords = sum(1 for t in doc if t.text in stop_words)
         return n_stopwords >= n
+
+    @staticmethod
+    def string_filter(doc: Doc, string: Optional[str] = None) -> bool:
+        """Method for filtering documents containing a specific string.
+
+        Args:
+            doc (Doc): SpaCy document
+            string (str, optional): String for filtering.
+        """
+        if string is None:
+            return True
+        else:
+            return string not in doc.text.lower()
```

Comment:
I would still appreciate an argument for adding this filter. I don't see it solving a task that isn't solved by existing filters.

Reply:
I think it is a great option to add a simple string-based filter. I, for one, would really like to see a test how two datasets compare, using a "lorem ipsum" filter and not using it. Furthermore, if we are creating a foundation for Danish language models, i think this is a free win to include, to let other people use string based filtering. If you truly think it makes no sense, I will remove it.

Reply:
Let us include it. I would very much welcome the comparison between filtering with and without lorem ipsum ;)